src/coreclr/tools/aot/Mono.Linker.Tests.Cases.Expectations/Assertions/BaseMemberAssertionAttribute.cs

```diff
@@ -12,4 +12,4 @@ namespace Mono.Linker.Tests.Cases.Expectations.Assertions
 	public abstract class BaseMemberAssertionAttribute : Attribute
 	{
 	}
-}
\ No newline at end of file
+}
```

Comment:
Can you please check if this should be "ported" to linker repo as well? (Probably the setting for lint to do this for us).

Reply:
Opened https://github.com/dotnet/linker/issues/3018 to track enabling runtime code-quality analyzer on top of existing linker analyzers